.github/workflows/changesets-release.yml

```diff
@@ -72,7 +97,10 @@ jobs:
   #         SLACK_WEBHOOK_URL: ${{ secrets.SLACK_RELEASE_NOTIFICATION_WEBHOOK_URL }}
 
   publish:
-    if: github.event.pull_request.merged == true
+    if: |
+      github.event.pull_request.merged == true &&
+      github.event.pull_request.user.login == 'neo4j-team-graphql' &&
+      startsWith(github.head_ref, 'changeset-release/')
```

Comment:
(It's on line 111)
Regarding the `needs` list, shouldn't it contain `aura-free-tests` and `aura-professional-tests` rather than `aura-tests`?

Reply:
Great spot!